app/components/login.jsx

```diff
@@ -1,202 +1,318 @@
-
-    this.props.join();
-  }
-
-  //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-  forgotPassword (e) {
-    //onsole.info("facebook");
-    e.preventDefault();
-
-    this.props['forgot-password']();
-  }
-
-  //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-  loginWithFacebook () {
-    // location.href = '/sign/in/facebook';
-    //onsole.info("loginWithFacebook");
-    Facebook.connect();
-  }
-
-  //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-  loginWithTwitter () {
-    location.href = '/sign/in/twitter';
-  }
-
-
-  stopPropagation(e){
-    e.stopPropagation();
- }
-
-  //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-  render () {
-    let classes = [ 'syn-login' ];
-
-    if ( this.props.show ) {
-      classes.push('syn--visible');
-    }
-
-    let content = (
-      <div onClick={this.stopPropagation.bind(this)}>
-      {(this.socialMeida)?(
-        <ButtonGroup block>
-          <Button medium primary onClick={ this.loginWithFacebook } className="login-with-facebook" ref='facebook'>
-            <Icon icon="facebook" />
-            <span className={ Component.classList(this) }> Facebook</span>
-          </Button>
-
-          <Button medium info onClick={ this.loginWithTwitter } className="login-with-twitter">
-            <Icon icon="twitter" />
-            <span> Twitter</span>
-          </Button>
-        </ButtonGroup>
-        ):null}
-
-        <div className="syn-form-group">
-          <label>Email</label>
-          <Input type='email' block autoFocus required medium placeholder="Email" ref="email" name="email" />
-        </div>
-
-        <div className="syn-form-group">
-          <label>Password</label>
-          <Input type='password' block required placeholder="Password" ref="password" medium name="password" />
-        </div>
-
-        <div className="syn-form-group syn-form-submit">
-          <Submit block large success radius>Login</Submit>
-        </div>
-
-        <Row>
-          <Column span="50" gutter className="text-left">
-            <p style={{margin: 0}}>Not yet a user?</p><a href="#" onClick={ this.signUp.bind(this) } ref='signup'>Sign up</a>
-          </Column>
-
-          <Column span="50" text-right gutter className="forgot-password">
-            <p style={{margin: 0}} className="forgot-password-label">Forgot password?</p>
-            <a
-              href              =   "#"
-              className         =   "forgot-password-link"
-              onClick           = { this.forgotPassword }
-            >
-            Click here
-            </a>
-          </Column>
-        </Row>
-      </div>
-    );
-
-    if ( this.state.info ) {
-      content = ( <Loading message="Loggin you in..." /> );
-    }
-
-    else if ( this.state.successMessage ) {
-      content = ( <div></div> );
+// https://github.com/EnCiv/civil-pursuit/issues/150
+
```

Comment:
My bad, I'm sorry. I didn't realize this would be writing over an existing component.  Can you rename the new component to sign-up.jsx and make the component SignUp

I don't want to interfere with the old code.

Reply:
Do you want me to retrieve the old login.jsx file? Or just keep it deleted to avoid confusion.

Reply:
Yes. So the old code will build and run.